keras_cv/layers/preprocessing/random_shear.py

```diff
@@ -151,3 +169,168 @@ def get_config(self):
             }
         )
         return config
+
+    def augment_bounding_boxes(self, bounding_boxes, transformation, **kwargs):
+        image = None
+        image = kwargs.get("image")
+        if self.bounding_box_format is None:
+            raise ValueError(
+                "`RandomShear()` was called with bounding boxes,"
+                "but no `bounding_box_format` was specified in the constructor."
+                "Please specify a bounding box format in the constructor. i.e."
+                "`RandomShear(bounding_box_format='xyxy')`"
+            )
+        height, width, _ = image.shape
+
+        bounding_boxes = keras_cv.bounding_box.convert_format(
+            bounding_boxes,
+            source=self.bounding_box_format,
+            target="xyxy",
```

Comment:
convert to target="rel_xyxy"

Reply:
augment transformation functions(_apply_horizontal_transformation_to_bounding_box, _apply_vertical_transformation_to_bounding_box) work on absolute coordinates hence the need to convert to xyxy